startup/gui/ocio.py

```diff
@@ -100,6 +100,7 @@ def __plugSet( plug ) :
 def __displayTransformCreator( name ) :
 
 	result = GafferImage.DisplayTransform()
+	result["channels"].setValue( "*[RGB]" )
```

Comment:
Thanks for spotting that mismatched channel mask - fixed in c80592b. @lucienfostier, yes, we only apply the display transforms to RGB channels, not auxiliary channels.